ANMN/temperature_grid_product/README.md

```diff
@@ -0,0 +1,28 @@
+# anmn_temp_gridded_product.py
+
+This piece of Python 2.7 code creates a gridded temperature products based on FV01 NetCDF files for a particular deployment_code:
+1- It retrieves a list of FV01 NetCDF files for a particular deployment_code from the WFS service of view anmn_ts_timeseries_map.
```

Comment:
This itemised list doesn't render properly. I think you need to use '.' instead of '-' after the numbers:
```md
1. It retrieves ...
2. Checks that ...
3. Considers ...
```